fluster/decoders/gstreamer.py

```diff
@@ -187,7 +187,12 @@ def gen_pipeline(
         output_filepath: Optional[str],
         output_format: OutputFormat,
     ) -> str:
-        caps = f"{self.caps} ! videoconvert dither=none ! video/x-raw,format={output_format_to_gst(output_format)}"
+        raw_caps = "video/x-raw"
+        try:
+            raw_caps += f",format={output_format_to_gst(output_format)}"
+        except Exception as error:
+            print(f"{error}")
```

Comment:
Can you apply the following changes? It makes the logs clearer.
Also please `import os` at the beginning of the file. Thanks
```suggestion
        except Exception as ex:
            print(f"WARNING: {os.path.basename(os.path.dirname(input_filepath))}, {ex}")
```